plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -554,10 +555,14 @@ def connection(self):
         # In previous versions, we relied on the functionality removed in
         # https://github.com/dbt-labs/dbt-core/pull/4062.
         if DBT_VERSION_TUPLE >= (1, 0):
-            adapter = get_adapter(self.dbt_config)
-            with adapter.connection_named("master"):
-                adapter.set_relations_cache(self.dbt_manifest)
```

Comment:
Why don’t we need this line anymore?

Reply:
No idea. @NiallRees, do you know what this line does, and whether it's still necessary? This code change was suggested by a user on dbt Slack. Let's see if it passes tests.

Reply:
The description here suggests it may be useful for performance. I'll add it back. 👍 🤷